samples/bluetooth/mesh/sensor_server/src/model_handler.c

```diff
@@ -718,6 +748,12 @@ static const struct bt_mesh_comp comp = {
 
 const struct bt_mesh_comp *model_handler_init(void)
 {
+#if IS_ENABLED(CONFIG_BT_MESH_COMP_PAGE_2)
```

Comment:
```suggestion
#if IS_ENABLED(CONFIG_BT_MESH_NLC_PERF_CONF)
```

Reply:
But at the same time, if someone want to use CONFIG_BT_MESH_COMP_PAGE_2 without CONFIG_BT_MESH_NLC_PERF_CONF. This will then be left out. That means the user have to use NLC not only comp page 2. That is why I would like to keep it as is.